makefiles/functions.Makefile

```diff
@@ -82,9 +82,9 @@ endef
 define publish_service_ssm
 	$(ROOT)/docker_run.py \
     	    --aws --dind -- \
-                wellcome/weco-deploy:2.0.0  \
+                wellcome/weco-deploy:3.1.0  \
                 --project-id="$(2)" \
-                --role-arn="$(DEV_ROLE_ARN)" \
+                -v \
```

Comment:
What does this `-v` flag do?

Reply:
[This](https://github.com/wellcomecollection/weco-deploy/blob/master/src/deploy/deploy.py#L66)?

Reply:
Verbosity! I think in CI this process should be verbose for easier debugging, at least right now.